src/solve.jl

```diff
@@ -75,7 +75,7 @@ function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algTyp
                                                        ode_prob,
                                                        OrdinaryDiffEq.alg_order(alg)))
     end
-    integrator.dt = dt
+    integrator.dt = zero(dt)
```

Comment:
>However, for all other steps it makes a huge difference

Not this line though, just the PR in general?

>So this specific line could be removed but I thought it would be preferable to keep dt, t, and tprev synchronized all the time.

Nah, it makes sense. Just was curious about the practical effect. In theory it would do a constant extrapolation if `dt=0`, I just was wondering if that ever came into play.